app/views/how-tos/switching-from-govuk-prototype-kit.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Switching from the GOV.UK prototype kit - NHS.UK prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1>
+        Switching from the GOV.UK prototype kit
+      </h1>
+
+
+<p>
+      If you have learned how to use the GOV.UK prototype kit, you will be able to use the NHS.UK prototype kit. This is because the two prototype kits are built in the same way. However, you must do some things differently.
+    </p>
+<h2>Installing and running the NHS.UK prototype kit</h2>
+
+<p>
+When you are installing and running your prototype:
+
+{{ list({
+  title: "Do",
+  type: "tick",
+  items: [
+    {
+      item: 'check when you install your prototype that you are using the right version of node.js - you may need to install and use a different version. If you need to make prototypes that for both NHS.UK and GOV.UK you can switch versions of node with a <a href="https://www.freecodecamp.org/news/node-version-manager-nvm-install-guide/">node version manager</a>'
+    },
+    {
+      item: 'start your local server by entering in the terminal: <pre class="app-pre"><code class="language-markup">npm run watch</code></pre> '
+    }
+  ]
+}) }}
+
+{{ list({
+  title: "Don't",
+  type: "cross",
+  items: [
+    {
+      item: 'start your local server the same way as in the GOV.UK prototype kit as it will not work'
```

Comment:
May as well be specific here to remind people of the difference?

```suggestion
      item: 'start your local server using `npm run dev` like with the GOV.UK prototype kit as it will not work'
```